quickwit/quickwit-metastore/src/metastore/postgresql_metastore.rs

```diff
@@ -564,59 +539,49 @@ impl Metastore for PostgresqlMetastore {
         Ok(())
     }
 
+    #[instrument(skip(self))]
     async fn list_indexes_metadatas(&self) -> MetastoreResult<Vec<IndexMetadata>> {
-        run_with_tx!(self.connection_pool, tx, {
-            let indexes: Vec<Index> = sqlx::query_as::<_, Index>("SELECT * FROM indexes")
-                .fetch_all(tx)
-                .await?;
-            let index_metadata: MetastoreResult<Vec<IndexMetadata>> = indexes
-                .into_iter()
-                .map(|index| index.index_metadata())
-                .collect::<MetastoreResult<_>>();
-            index_metadata
-        })
+        let pg_indexes = sqlx::query_as::<_, postgresql_model::Index>("SELECT * FROM indexes")
+            .fetch_all(&self.connection_pool)
+            .await?;
+        pg_indexes
+            .into_iter()
+            .map(|pg_index| pg_index.index_metadata())
+            .collect()
     }
 
-    #[instrument(skip(self),fields(index_id=index_metadata.index_id.as_str()))]
+    #[instrument(skip(self),fields(index_id=index_metadata.index_id))]
```

Comment:
If we're adding some formatting/whitespace to the instrument macros while we go, we should probably add them to any of the other instruments we add here.

```suggestion
    #[instrument(skip(self), fields(index_id=index_metadata.index_id))]
```